kits/real-estate-analyst/agent.md

```
@@ -0,0 +1,52 @@
+# Real Estate Investment Analyst
+
+## Overview
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)